packages/kbn-es/src/cluster.ts

```diff
@@ -437,17 +438,20 @@ export class Cluster {
         });
       }
 
-      // once the cluster is ready setup the native realm
-      if (!skipNativeRealmSetup) {
+      // once the cluster is ready setup the realm
+      if (!skipSecuritySetup) {
         const nativeRealm = new NativeRealm({
           log: this.log,
           elasticPassword: options.password,
           client,
         });
 
         await nativeRealm.setPasswords(options);
-      }
 
+        if (args.some((arg) => arg.includes(MOCK_IDP_REALM_NAME))) {
+          await ensureSAMLRoleMapping(client);
+        }
+      }
```

Comment:
I originally planned to make a call to ES and pass roles from `roles.yml` explicitly, but we already have a function in '@kbn/mock-idp-utils' that we use for serverless. Re-using for consistency.